src/styles/_footer.scss

```diff
@@ -107,8 +116,8 @@
     padding: units(3) 0;
   }
 
-  .usa-footer__primary-link {
-    padding: units(1.5);
+  .usa-link.usa-footer__primary-link {
+    padding: units(1.5) 0 0 0;
```

Comment:
Are you trying to override padding here?

Reply:
What do you mean by override? There's bigger padding in mobile/tablet view, so the tap area is bigger. I guess I could have more bottom padding in desktop as well. I'd just have to reposition the underline

Reply:
updated

Reply:
By override, I meant that there is a default padding, and you're overriding it by adding 0s. This is fine.

Reply:
Okay - yeah it's not overriding the default - it's just changing between the mobile and desktop views